components/01-atoms/buttons/_buttons.scss

```diff
@@ -8,6 +10,15 @@ $button-outline-weights: (
   '4': var(--border-thickness-4),
 );
 
+// The button-reset mixin is used to remove browser default styles for button
+// elements that shouldn't look like a traditional "button".
+@mixin button-reset {
```

Comment:
Similar to `list-reset` comment I had earlier. Wouldn't this be namespaced during implementation to `button.reset`? On my local I was able to:
1. Change this to `@include reset  { ....` 
2. Add `@use '../../../01-atoms/buttons/buttons' as button;` at the top of the `_menu-toggle.scss` file
3. Swap out the mixin reference to `@include button.reset;`

Reply:
I did this in the Breadcrumbs PR when I refactored all of the `@import`s to `@use`s. https://github.com/yalesites-org/component-library-twig/pull/31/files#diff-e6b4386e20a680dd5b7e303065b8387277192ebb6940cc14cfe6d3a3b1e1f113